external/mlir-hal/include/mlir/Dialect/MHAL/IR/MHALAttrDefs.td

```diff
@@ -79,4 +79,15 @@ def MHAL_KernelPackageAttr : MHAL_Attr<"KernelPackage"> {
     let hasCustomAssemblyFormat = 1;
 }
 
+// Misc Attrs
+def MHAL_PrefillAttr : MHAL_Attr<"Prefill"> {
+  let mnemonic = "mhal.prefill";
+
+  let parameters = (ins
+    "uint32_t":$argIndex,
+    "uint32_t":$initValue
```

Comment:
`"TypedAttribute": $initValue` so we don't lose type information. MIGraphX will want it.

Reply:
@krzysz00 , are you sure that there is something like `TypedAttribute`. I search over `llvm-project/mlir` and I couldn't find something similar

Reply:
There is `NamedAttribute`

Reply:
Maybe it is `TypedAttrInterface` ?

https://github.com/llvm/llvm-project/blob/bf716fb7169b500a7fe3f20a39ac0288fb13cac5/mlir/include/mlir/Dialect/Arith/IR/ArithOps.td#L180-L186

Reply:
I guess I found. Let me try `TypedAttr`

Reply:
@krzysz00, `TypedAttr` worked and it resulted in less code. Thanks!